components/campus-toggle/campus-toggle.test.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import CampusToggle from "./campus-toggle.component";
+import renderer from "react-test-renderer";
+
```

Comment:
For this unit test file, the methods of the components should also be tested. if they depend on external things, mocks should be created for those dependencies

Reply:
This point will be addressed in a later task. Since function testing will require Enzyme.

https://enzymejs.github.io/enzyme/docs/guides/react-native.html